biz.aQute.remote/test/biz/aQute/remote/AgentTest.java

```diff
@@ -185,10 +185,11 @@ public void testAgentUpdateBundleFromURL() throws Exception {
 	}
 
 	public void testAgentShell() throws Exception {
-		String result = supervisor.getAgent().shell("lb");
-		assertNotNull(result);
-		assertTrue(result.contains("START"));
-		assertTrue(result.contains("LEVEL"));
+		// This test requires a gogo to be added to the framework
```

Comment:
If this method is not useful, comment out or remove then entire method. Having an empty test method inflates the test numbers.